pykickstart/commands/autopart.py

```diff
@@ -209,6 +288,13 @@ def _getParser(self):
         return op
 
 class F17_AutoPart(F16_AutoPart):
+    """
+        ``autopart [--encrypted [--passphrase=<phrase>]
+        [--escrowcert=<url> [--backuppassphrase=<passwd>]]]
+        [--nolvm] | [--type=<type>]``
+
+        where <type> is lvm, btrfs, plain
```

Comment:
I'd like to see this look more like a sentence:

Where <type> is one of lvm, btrfs, or plain